Vagrantfile

```diff
@@ -0,0 +1,68 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+INVENTORY_FILE = "inventory.vagrant"
+MEMORY_DEFAULT = 384
+HOSTS = [
+  { name: 'jenkins', ip: '172.18.10.9',
+    roles: %w{jenkins-master} },  
+]
+
+def generate_inventory(hosts, file)
+  roles = {}
+
+  File.open(file, "w") do |f|
+    hosts.each do |host|
+      host_string = "#{host[:name]} \
+internal_ip=#{host[:ip]} \
+external_ip=#{host[:ip]}.xip.io \
+ansible_ssh_host=#{host[:ip]}\n"
+
+      f.write(host_string)
+      host[:roles].each do |role|
+        roles[role] ||= []
+        roles[role] << host_string
+      end
+    end
+
+    roles.each do |role, hosts|
+      f.write("\n[#{role}]\n")
+      f.write(hosts.join)
+    end
+  end
+end
+
+generate_inventory(HOSTS, INVENTORY_FILE) if %w{up provision}.include?(ARGV[0])
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.ssh.private_key_path = File.expand_path('~/.vagrant.d/insecure_private_key')
+  config.ssh.insert_key = false
+
+
+  HOSTS.each do |host|
+    config.vm.define host[:name] do |c|
+      c.vm.hostname = host[:name]
+      c.vm.network :private_network, ip: host[:ip]
+
+      c.vm.provider :virtualbox do |v|
+        v.memory = host[:memory] || MEMORY_DEFAULT
+      end
+
+      c.vm.provider :vmware_fusion do |v|
+        v.vmx["memsize"] = host[:memory] || MEMORY_DEFAULT
+      end
+    end
+  end
+
+  config.vm.provision :shell, inline: "apt-get purge -qq -y --auto-remove chef puppet"
+
+  config.vm.define HOSTS.last[:name] do |host|
```

Comment:
We can remove this out block. It was a hack in alphagov/tsuru-ansible@9508525e6d18dce09576fb286c22b4dc313d8cd8 to account for ordering where we had mutlple hosts that depended on one another.